pkgs/servers/owncast/default.nix

```diff
@@ -1,43 +1,34 @@
-{ lib, buildGoModule, fetchFromGitHub, nixosTests, bash, which, ffmpeg, makeWrapper, coreutils, ... }:
-
-buildGoModule rec {
+{
+  lib
+  ,buildGoModule
+  ,callPackage
+  ,fetchFromGitHub
+  ,nixosTests
+  ,ffmpeg
+  ,makeWrapper
+  ,system
+}:
+
+let
+  version = "0.1.1";
+in buildGoModule rec {
   pname = "owncast";
-  version = "0.0.13";
-
+  inherit version;
```

Comment:
```suggestion
  version = "0.1.1";

```

Reply:
I think it's left over from when I didn't have a `rec`... 

I thought that `let.. in` was preferred to `rec`? have removed the latter :)